kDrive/AppDelegate.swift

```diff
@@ -197,152 +163,11 @@ final class AppDelegate: UIResponder, UIApplicationDelegate, AccountManagerDeleg
-
-            // reset the shortcut item
-            shortcutItemToProcess = nil
-        }
-    }
-
-    func refreshCacheScanLibraryAndUpload(preload: Bool, isSwitching: Bool) {
-        Log.appDelegate("refreshCacheScanLibraryAndUpload preload:\(preload) isSwitching:\(preload)")
-
-        guard let currentAccount = accountManager.currentAccount else {
-            Log.appDelegate("No account to refresh", level: .error)
-            return
-        }
-
-        let rootViewController = window?.rootViewController as? UpdateAccountDelegate
-
-        availableOfflineManager.updateAvailableOfflineFiles(status: ReachabilityListener.instance.currentStatus)
-
-        Task {
-            do {
-                let oldDriveId = accountManager.currentDriveFileManager?.drive.objectId
-                let account = try await accountManager.updateUser(for: currentAccount, registerToken: true)
-                rootViewController?.didUpdateCurrentAccountInformations(account)
-
-                if let oldDriveId,
-                   let newDrive = driveInfosManager.getDrive(primaryKey: oldDriveId),
-                   !newDrive.inMaintenance {
-                    // The current drive is still usable, do not switch
-                    scanLibraryAndRestartUpload()
-                    return
-                }
-
-                let driveFileManager = try accountManager.getFirstAvailableDriveFileManager(for: account.userId)
-                accountManager.setCurrentDriveForCurrentAccount(drive: driveFileManager.drive)
-                showMainViewController(driveFileManager: driveFileManager)
-                scanLibraryAndRestartUpload()
-            } catch DriveError.NoDriveError.noDrive {
-                let driveErrorNavigationViewController = DriveErrorViewController.instantiateInNavigationController(
-                    errorType: .noDrive,
-                    drive: nil
-                )
-                setRootViewController(driveErrorNavigationViewController)
-            } catch DriveError.NoDriveError.blocked(let drive), DriveError.NoDriveError.maintenance(let drive) {
-                let driveErrorNavigationViewController = DriveErrorViewController.instantiateInNavigationController(
-                    errorType: drive.isInTechnicalMaintenance ? .maintenance : .blocked,
-                    drive: drive
-                )
-                setRootViewController(driveErrorNavigationViewController)
-            } catch {
-                UIConstants.showSnackBarIfNeeded(error: DriveError.unknownError)
-                Log.appDelegate("Error while updating user account: \(error)", level: .error)
-            }
-        }
-    }
-
-    private func scanLibraryAndRestartUpload() {
-        // Resolving an upload queue will restart it if this is the first time
-        @InjectService var uploadQueue: UploadQueue
+    // Migrated to sceneWillEnterForeground
+    // func applicationWillEnterForeground(_ application: UIApplication) { }
```

Comment:
```suggestion
```

Reply:
Yeah correct, no need for dead commented code.